components/brave_vpn/browser/connection/win/brave_vpn_helper/brave_vpn_helper_crash_reporter_client.cc

```diff
@@ -54,12 +54,11 @@ void BraveVPNHelperCrashReporterClient::InitializeCrashReportingForProcess(
   }
   install_static::InitializeProductDetailsForPrimaryModule();
   crash_reporter::SetCrashReporterClient(instance);
-  std::wstring user_data_dir;
-  install_static::GetUserDataDirectory(&user_data_dir, nullptr);
 
   crash_reporter::InitializeCrashpadWithEmbeddedHandler(
```

Comment:
auto upload needs to be dependent on user settings for Brave crash upload

Reply:
it works via https://github.com/brave/brave-core/blob/master/components/brave_vpn/browser/connection/win/brave_vpn_helper/brave_vpn_helper_crash_reporter_client.cc#L142